packages/vaadin-virtual-list/src/vaadin-virtual-list.js

```diff
@@ -128,10 +128,27 @@ class VirtualListElement extends ElementMixin(ThemableMixin(PolymerElement)) {
   /** @private */
   __updateElement(el, index) {
     if (this.renderer) {
+      if (el.__renderer !== this.renderer) {
```

Comment:
Seems that it doesn't clear the content after the renderer has been removed altogether.

As I remember, the content is cleared after the renderer is removed in other components.

Reply:
👍  Fixed and added a test